Index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>
+   Apple.com Old Site
+  </title>
+  <meta charset="UTF-8" />
+  <link href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&amp;subset=latin" rel="stylesheet">
+
+  <meta name="description" content="Old Apple.com's Home Page Clone Project Made by Prabhakar and Alex" />
+  <meta name="keywords" content="HTML, CSS, Old Apple.com Home Page Clone Project, The Odin Project" />
+  <meta name="author" content="Prabhakar and Alex" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="CSS/style.css" rel="stylesheet" />
+</head>
+<body>
+
+<header class="addRow">
```

Comment:
**[OPTIONAL]** I think you are not having a good semantic HTML structure you need to use:
- header then main then footer, conceded using that.

Reply:
Thanks for suggesting! Elements were too few so main was vomited but now added in the lower section. Thanks!